gp-includes/assets-loader.php

```diff
@@ -17,6 +17,7 @@ function gp_register_default_styles() {
 	// Register our base style.
 	wp_register_style( 'gp-base', $url . '/style' . $suffix, array(), filemtime( $path . 'style' . $suffix ) );
 	wp_register_style( 'gp-jquery-webui-popover', $url . '/jquery.webui-popover' . $suffix, array( 'gp-base' ), filemtime( $path . 'jquery.webui-popover' . $suffix ) );
+	wp_register_style( 'driver-js', 'https://cdn.jsdelivr.net/npm/driver.js@1.0.1/dist/driver.css', array(), '1.0.1' );
```

Comment:
I think we should be using a local copy, so can you add this file under `assets/css` to the PR and use that?